project-base/scripts/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+numberRegex='^[0-9]+([.][0-9]+)?$'
+operatingSystem=''
+allowedValues=(1 2)
+declare -A operatingSystemsByIndex
+
+operatingSystemsByIndex[1]='Linux'
+operatingSystemsByIndex[2]='Mac'
+
+echo This is installation script that will install demo Shopsys Framework application on docker with all required containers and with demo database created.
+
+echo "Start with specifying your operating system: \
+    1) Linux
+    2) Mac
+    "
+
+while [[ 1 -eq 1 ]]
+do
+    read -p "Enter OS number: " operatingSystem
```

Comment:
This step can be fully automated using `uname` command, which outputs either `"Darwin"` on a Mac or `"Linux"` on Linux.

We have implemented a similar thing in monorepo-tools where used flags differ between the OSs (introduced in #282):
https://github.com/shopsys/shopsys/blob/55e785de1ba7861ebab066acb1d14e595b336b15/packages/monorepo-tools/rewrite_history_from.sh#L38-L44

Reply:
be aware that for windows we recommend to develop in WSL and uname for it is also Linux https://travis-ci.org/boris-brtan/test-travis-os/builds/483328068 but it should behave osx-like

Reply:
Yeah I totally forgot about WSL. Ok but anyway, I would like to stay with filling the number of OS since windows will output "Linux" during `uname`..

Feels like not big of a deal? Or do you have any idea on how to make this fully automatic, discovering the windows OS on WSL?

Reply:
Ok, for me but to answer the question:
```
uname
-r, --kernel-release     print the kernel release
-v, --kernel-version     print the kernel version
```
returns Microsoft word https://travis-ci.org/boris-brtan/test-travis-os/jobs/483478869
```
4.4.0-17134-Microsoft
#523-Microsoft Mon Dec 31 17:49:00 PST 2018
```
Or there exists `/etc/wsl.conf` file.

Reply:
That seems little bit sketchy for me. I would prefer to let fill out the OS number.